apps/sms/test/marionette/lib/messages.js

```diff
@@ -87,6 +88,8 @@
       var actions = client.loader.getActions();
 
       return {
+        client: client,
```

Comment:
I agree, this looks like a hack. The `client` is needed for the higher-level pages that initialize an other page. As this is linked to the initialization of subpages, let's move this discussion on general thread.